Misc/NEWS.d/next/Library/2018-05-23-17-07-54.bpo-33625.nzQgD8.rst

```diff
@@ -0,0 +1,2 @@
+Release GIL on `grp.getgrnam`, `grp.getgrgid`, `pwd.getpwnam` and
+`pwd.getpwuid`. Patch by William Grzybowski.
```

Comment:
The NEWS entry is not correct. The GIL is not *always* released. It depends if reentrant variants of these functions are available.